mm2src/coins/coin_balance.rs

```diff
@@ -15,19 +13,20 @@ use std::ops::Range;
 use std::{fmt, iter};
 
 pub type AddressIdRange = Range<u32>;
+pub(crate) type HDBalanceAddress<T> = <<T as HDWalletBalanceOps>::HDAddressScanner as HDAddressBalanceScanner>::Address;
 
 pub enum EnableCoinBalanceError {
     NewAddressDerivingError(NewAddressDerivingError),
-    NewAccountCreatingError(NewAccountCreatingError),
+    NewAccountCreatingError(NewAccountCreationError),
```

Comment:
It's worth changing `Creating` to `Creation` here too 🙂

Reply:
done